bootcompiler/src/main/scala/org/mozartoz/bootcompiler/transform/TailCallMarking.scala

```diff
@@ -0,0 +1,48 @@
+package org.mozartoz.bootcompiler
+package transform
+
+import ast._
+import oz._
+import symtab._
+
+object TailCallMarking extends Transformer {
+  
+  def markTailCalls(name: Symbol, statement: Statement): Statement = statement match {
+    
+    case call @ CallStatement(callable @ Variable(sym), args) =>
+      treeCopy.TailMarkerStatement(statement, call)
+
+    case CompoundStatement(stats) =>
```

Comment:
The whitespace style seems to be `only after a case body, but not before a def or the first case`.
Could you make it consistent to what it is in `Desugar`?